app/controllers/helper/ShapefilesCreatorHelper.java

```diff
@@ -0,0 +1,443 @@
+package controllers.helper;
+
+import java.io.*;
+import java.util.*;
+import java.util.zip.*;
+import scala.Option;
```

Comment:
Can you make sure we need all these imports as well?

Reply:
unfortunately all of those imports are necessary, though i was able to replace some with .* imports so it takes fewer lines